graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/LoadBalancerStatusResource.java

```diff
@@ -55,16 +55,40 @@ public LoadBalancerStatusResource(ServerStatus serverStatus) {
         this.serverStatus = serverStatus;
     }
 
+    /**
+     * Implementing StatusType for 429 TOO MANY REQUEST (the most suitable HTTP status code for
+     * requesting throttling), because javax.ws.rs.core.Response.Status does not implement it.
+     */
+    private class TooManyRequestsStatus implements Response.StatusType {
```

Comment:
Please move this as a self-contained class into the `org.graylog2.rest` package.